browser/js/common/directives/graphs/add-graph.directive.html

```diff
@@ -0,0 +1,33 @@
+<section>
+	<form name="submitLineGraph" role="form" novalidate>
+	    <fieldset>
+			<div class="row">
+			  <div class="form-group col-xs-8">
+			  	<div class="row">
+		          <div class="form-group col-xs-12">
+		            <label for="column1" class="control-label">Column #1</label>
+		            <div>
+		              <select ng-model="column1.name"  name="column1" class="form-control" ng-options="col for col in columns"></select>
+		            </div>
+		          </div>
+		          <div class="form-group col-xs-12">
```

Comment:
Maybe a good spot for an `ng-repeat`? I know there are only two of them, but still. Something like `<div ng-repeat="selectedColumn in selectedColumns">...<input ng-model="selectedColumn.name">...</div>` and then in the controller/link for this directive `$scope.selectedColumns = [{}, {}];`.